helm-chart/README.md

```diff
@@ -39,6 +39,9 @@ helm show values gatling/enterprise-locations-packages > values.yaml
 > [!TIP]
 > When connecting to the cluster using HTTPS, if a custom truststore and/or keystore is needed, `KUBERNETES_TRUSTSTORE_FILE`, `KUBERNETES_TRUSTSTORE_PASSPHRASE` and/or `KUBERNETES_KEYSTORE_FILE`, `KUBERNETES_KEYSTORE_PASSPHRASE` environment variables should be set.
 
+> [!TIP]
+> The control plane and location configurations allow for the setup of a reverse proxy. To enable this, uncomment the extraContent section and its associated values in the configuration file for the control plane and/or location. Ensure the reverse proxy is configured to rewrite the Host header to `api.gatling.io` to ensure proper functionality.
```

Comment:
Actually, the correct term is "forward proxy"

Reply:
Thanks for forwarding!